src/main/resources/json-schema/digital_entity.json

```diff
@@ -0,0 +1,163 @@
+{
+  "$id": "digitalEntity",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "properties": {
+    "ods:id": {
+      "type": "string",
+      "description": "The unique digital identifier of the object",
+      "pattern": "https:\/\/doi.org\/10.22\/(.){3}-(.){3}-(.){3}",
```

Comment:
Should this have the handle pattern instead for now?

Reply:
To clarify: yes media objects will have DOIs, but we're not there YET.

Reply:
Sure with the schema's I was aiming for how we would like to have things in the end so we won't have to add/change them continuously but as DOIs for media might take a bit put it on handle for now